core/src/main/se/tevej/game/model/components/PositionComponent.java

```diff
@@ -3,27 +3,43 @@
 import com.badlogic.ashley.core.Component;
 
 public class PositionComponent implements Component {
-    private float x;
-    private float y;
+    private int x;
+    private int y;
 
-    public PositionComponent(float x, float y) {
+    public PositionComponent(int x, int y) {
         this.x = x;
         this.y = y;
     }
 
-    public void setX(float x) {
+    public void setX(int x) {
         this.x = x;
     }
 
-    public void setY(float y) {
+    public void setY(int y) {
         this.y = y;
     }
 
-    public float getX() {
+    public int getX() {
         return x;
     }
 
-    public float getY() {
+    public int getY() {
         return y;
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (obj instanceof PositionComponent) {
```

Comment:
Ugh... But ok.

Reply:
Kind of had to, for the contains method.